server/spec/features/openstudio_urbanopt_algo_spec.rb

```diff
@@ -105,7 +105,7 @@
     ]
 
     # run an analysis
-    command = "#{@bundle_cmd} #{@meta_cli} run_analysis --debug --verbose '#{@project}/URBANopt_singlerun.json' 'http://#{@host}' -z 'URBANopt_NSGA' -a single_run"
+    command = "#{@bundle_cmd} #{@meta_cli} run_analysis --debug --verbose '#{@project}/UrbanOpt_singlerun.json' 'http://#{@host}' -z 'UrbanOpt_NSGA' -a single_run"
```

Comment:
these are getting changed to URBANopt to match the trademarks or whatever.  thats in another PR, but not really an issue here I suppose since these tests are not running right now.

Reply:
Tests are not running right now but we'll enable them as soon as the UrbanOpt gems are released. I guess we resolve any conflicts if there are any when that time comes.